app/dashboard/views.py

```diff
@@ -710,6 +710,16 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0, stdbounties_id=None
     return TemplateResponse(request, 'bounty_details.html', params)
 
 
+def quickstart(request):
```

Comment:
I'm adding `staff_member_required` to the new views until we're ready to launch `quickstart`